data/vamsikurimeti.json

```
@@ -0,0 +1,45 @@
+{
+  "name": "Mohan Vamsi Kurimeti",
+  "bio": "🚀 Fullstack Developer 🌐 | Node.js, Angular, GCP, AWS, Spring Boot | Docker enthusiast 🐳 | GitHub aficionado 💻 | Crafting exceptional digital experiences with style and precision ✨ Let's collaborate and build amazing things together! 🤝"
+"avatar":"https://github.com/vamsikurimeti.png"
+"tags":["Javascript", "Typescript", "Node JS", "Angular", "Primeng","Bootstrap", "Prime Flex", "Springnoot", "Docker", "Kubernetes","Networking","OWASP", "Cybersecurity", "Burpsuite", "Kali Linux", "N map","GCP", "AWS","Github", "Opensource"
```

Please add the missing commas `,` & braces to validate your `json`. Also, remove the `avatar` key as we no longer need that. 

```suggestion
  "bio": "🚀 Fullstack Developer 🌐 | Node.js, Angular, GCP, AWS, Spring Boot | Docker enthusiast 🐳 | GitHub aficionado 💻 | Crafting exceptional digital experiences with style and precision ✨ Let's collaborate and build amazing things together! 🤝",
"tags":["Javascript", "Typescript", "Node JS", "Angular", "Primeng","Bootstrap", "Prime Flex", "Springnoot", "Docker", "Kubernetes","Networking","OWASP", "Cybersecurity", "Burpsuite", "Kali Linux", "N map","GCP", "AWS","Github", "Opensource"],
```